src/librustc/middle/region.rs

```diff
@@ -371,6 +371,19 @@ struct RegionResolutionVisitor<'tcx> {
 
     // The number of expressions and patterns visited in the current body
     expr_and_pat_count: usize,
+    // When this is 'true', we record the Scopes we encounter
```

Comment:
```suggestion
    // When this is `true`, we record the `Scopes` we encounter
```
Generally, identifiers are surrounded in backticks in comments (here and elsewhere).